code/src/components/HandleTasks.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { todos } from "reducers/todos";
+
+import styled from "styled-components";
+import { Button } from "lib/Buttons";
```

Comment:
Clean import, good! I think Maks said in one lecture that the imports should be structured like this: 
1. Imports from "outside" like React, useSelector, useDispatch and so on.
2. Imports from our own code like { todos }, components etcetera.
3. Imports with styling.

Just good to have in mind in the future :)

Reply:
Thank you, Johanna!  I'll change this :)))